roles/edpm_nova/tasks/configure.yml

```diff
@@ -15,6 +15,8 @@
   loop:
     - {"path": "{{ edpm_nova_config_dest }}", "mode": "0755"}
     - {"path": "/var/lib/openstack/config/containers", "mode": "0755"}
+    - { "path": "{{edpm_nova_cacerts_dest}}" }
```

Comment:
These are also gong to need permissions.

Reply:
done